tests/system/feature_store/test_spark_engine.py

```diff
@@ -543,6 +548,11 @@ def test_aggregations(self):
             "bid": {("moshe", "cohen"): 12, ("yosi", "levi"): 16},
             "bid_sum_1h": {("moshe", "cohen"): 2012, ("yosi", "levi"): 37},
             "bid_max_1h": {("moshe", "cohen"): 2000, ("yosi", "levi"): 16},
+            "bid_sqr_1h": {("moshe", "cohen"): 4000144, ("yosi", "levi"): 477},
```

Comment:
I'm not able to find a computation that fits all these results - please clarify
"moshe" values are {2000,12} with aggr:
```
sqr = 2000^2 = 12^2 = 4000144    // fits computation for assumed 2 values
avr = (2000 + 12) / 2 = 1006
variance  = (2000-1006)^2 + (12-1006)^2 = 1976072 // fits computation if we assume we sum the square of avg-distance (without dividing by count)
```
"yossi" values are {10,11,16} with aggr:
```
sqr = 10^2 + 11^2 + 16 ^2 = 477 // fits computation, so assumed 3 values are correct
avr = (10 + 11 + 16) / 3 = 12.33
variance  = (10-12.33)^2 + (11-12.33)^2 +(16-12.33)^2 = 20.66 // Not equal 10.33 in computation (and why /2, if avg should be /3) 
```

Reply:
https://github.com/mlrun/storey/blob/v1.3.7/storey/aggregation_utils.py#L98
```
(count * sqr - sum * sum) / (count * (count - 1))
```
(sqr = sum of squares)

Reply:
Btw, I think it's very unlikely that the long-standing storey calculation and the new spark implementation are both wrong in the same way, so I didn't even look into it once I saw that they match.